.github/workflows/someip2val_build.yml

```diff
@@ -19,6 +19,7 @@ on:
   pull_request:
     paths:
       - ".github/workflows/someip2val_build.yml"
+      - ".github/workflows/check_push_rights.yml"
       - "someip2val/**"
```

Comment:
what about - ".github/actions/post-container-location/action.yml"? included in the other ones but here not

Reply:
Yes, at the moment someip2val does not use that action, so no need to depend on it. We could potentially try to align how we use thata ction when/if we add ddds2val docker push to ttl/ghcr as part of CI

Reply:
Okay, just wanted to confirm that it is wanted :)